hubble-relay/Dockerfile

```diff
@@ -6,12 +6,24 @@ ARG UBUNTU_IMAGE=quay.io/cybozu/ubuntu:20.04
 FROM ${GOLANG_IMAGE} as build
 
 COPY TAG /
+COPY 17930.patch /tmp/
+COPY 18454.patch /tmp/
+COPY 19936.patch /tmp/
+COPY 20006.patch /tmp/
+COPY 20092.patch /tmp/
+COPY 20176.patch /tmp/
 
 # LICENSE.all
 WORKDIR /go/src/github.com/cilium/cilium
 RUN VERSION=$(cut -d \. -f 1,2,3 < /TAG ) \
     && curl -fsSL "https://github.com/cilium/cilium/archive/v${VERSION}.tar.gz" | \
       tar xzf - --strip-components 1 \
+    && patch -p1 --no-backup-if-mismatch < /tmp/17930.patch \
+    && patch -p1 --no-backup-if-mismatch < /tmp/18454.patch \
+    && patch -p1 --no-backup-if-mismatch < /tmp/20006.patch \
+    && patch -p1 --no-backup-if-mismatch < /tmp/20092.patch \
+    && patch -p1 --no-backup-if-mismatch < /tmp/19936.patch \
+    && patch -p1 --no-backup-if-mismatch < /tmp/20176.patch \
```

Comment:
`hubble-relay` doesn't seem like it needs these patches applied. Are you applying the same patches as for `cilium` and `cilium-operator` simply as a precaution?

Reply:
You're right. hubble-relay doesn't need these patches. I will remove those.

I thought parser.go change might affect hubble-relay, but that is for the local hubble(cilium-agent).
https://github.com/cilium/cilium/commit/ccd4a8ec08565b61034dd32262971e5eb6634d60#diff-42f10ee9d576ca64a496b2b00bbd41936aba49e669b4b44b27bb44d676694ecb